.github/workflows/build.yml

```diff
@@ -105,9 +106,26 @@ jobs:
     - name: Make build directory
       run: mkdir build
 
+#    - name: Get Conan
+#      run: |
+#        pip3 install wheel setuptools
+#        pip3 install conan --upgrade
+#        mkdir build
+        
+#    - name: Fix Path on Ubuntu
+#      if: startsWith(matrix.config.os, 'ubuntu')
+#      run: echo "/home/runner/.local/bin" >> $GITHUB_PATH
+
+    - name: Clean Linux Clang dependencies
+      if: startsWith(matrix.config.os, 'ubuntu')
+      run: |
+        sudo  apt list --installed "libclang*-dev"
+        sudo apt-get purge --auto-remove libclang-common-10-dev
+        sudo apt-get purge --auto-remove libclang-common-11-dev
+
     - name: Add Linux dependencies
       if: startsWith(matrix.config.os, 'ubuntu')
-      run: sudo apt-get install libnss3-dev libdbus-1-dev
+      run: sudo apt-get install libnss3-dev libdbus-1-dev libclang-12-dev libxshmfence-dev nodejs
```

Comment:
You just need to run this line

Reply:
````
$ sudo apt-get install libnss3-dev libdbus-1-dev libclang-12-dev libxshmfence-dev nodejs
Reading package lists... Done
Building dependency tree       
Reading state information... Done
libxshmfence-dev is already the newest version (1.3-1).
nodejs is already the newest version (10.19.0~dfsg-3ubuntu1).
libdbus-1-dev is already the newest version (1.12.16-2ubuntu2.1).
libnss3-dev is already the newest version (2:3.49.1-1ubuntu1.5).
libclang-12-dev is already the newest version (1:12.0.0-3ubuntu1~20.04.4).
0 upgraded, 0 newly installed, 0 to remove and 1 not upgraded.
````

Reply:
Build log from my local machine: https://gist.github.com/patricia-gallardo/c1c0a559ded9e67364c4ad1964423275

Reply:
I'm guessing the test for the presence of libnss3 is wrong?

Reply:
Indeed this is surprising. Maybe try to install apt's pkgconfog ?

Reply:
Like this?
````
$ sudo apt install pkg-config
[sudo] password for patricia-gallardo: 
Reading package lists... Done
Building dependency tree       
Reading state information... Done
pkg-config is already the newest version (0.29.1-0ubuntu4).
0 upgraded, 0 newly installed, 0 to remove and 1 not upgraded.
````

Reply:
Yes. I have no idea why it is not detected. I have to check how it does the detection, because the log does not says anything. Maybe tomorrow

Reply:
This is outside my area of expertise :grimacing: sorry - this is output from my machine

````
$ pkg-config --modversion nss
3.49.1
$ pkg-config --libs "nss >= 3.50"
Requested 'nss >= 3.50' but version of NSS is 3.49.1
$ pkg-config --libs "nss >= 3.26"
-lnss3 -lnssutil3 -lsmime3 -lssl3 -lplds4 -lplc4 -lnspr4
````

Reply:
Added this to my local environment since I saw that you had added it to the workflow and then it found it
````
PKG_CONFIG_PATH=/usr/lib/x86_64-linux-gnu/pkgconfig/
````